app/dashboard/templates/profiles/tab_follow.html

```diff
@@ -0,0 +1,145 @@
+{% load i18n static %}
+
+{% if not hidden %}
+<div id="profile-tabs" class="tab-container font-body mt-3 mb-4">
+  <button type="button" id="nav-description" href="{{profile.url}}/follow?sub=followers" class="text-center section-tab {% if foltab == "followers" %} active {% endif %}">
+    {% trans "FOLLOWERS" %} <span  class="nav-badge">({{followers.count}})</span>
+  </button>
+  <button type="button" id="nav-description" href="{{profile.url}}/follow?sub=following" class="text-center section-tab {% if foltab == "following" %} active {% endif %}">
+    {% trans "FOLLOWING" %} <span  class="nav-badge">({{following.count}})</span>
+  </button>
+</div>
+{% endif %}
+
+<div class="tab-projects d-flex flex-column">
+  {% if foltab == 'followers' %}
+    <!-- Show This users followers-->
+    {% for user in followers %}
+      <div class="tab-projects__item d-flex mb-0 pt-1 pb-1 bounty_row">
+        <div class="avatar-container col-1 justify-content-center hide_min_viewport">
+          <img class="profile-header__avatar mr-3" src="/dynamic/avatar/{{ user.profile.handle }}" alt="">
+        </div>
+        <div class="tab-projects__item-details col-3 d-flex flex-column bounty-detail">
+          <div class="d-flex flex-column">
+            <div class="font-weight-semibold font-header">{{user.profile.name}}</div>
+            <div class="text-highlight-gc-blue  font-smaller-1">{{user.profile.handle}}</div>
+          </div>
+        </div>
+        <div class="mb-0 mt-1 col-5 d-flex flex-row align-items-center">
+        <div class="mr-5"  style="height: fit-content">
+          <button class="btn btn-outline-gc-blue btn-sm flex-grow-1 font-smaller-5 position-relative quick-link" data-openchat="{{user.profile}}">
+            <i class="fas fa-comment-dots" data-placement="bottom" data-toggle="tooltip" data-html="true"
```

Comment:
lets move the tooltip to the button to show it on all the button hover not just the icon

Reply:
done